### PR TITLE
Irminger 9 and Papa 9a Telemetered ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
The directories for GI05MOAS-PG575 and GI05MOAS-GL538 are not on the Raw Data Repo yet. I have a request into CI to create them for us so the Raw Data can appear. The directories are present on ciw so ingestion can proceed without the Raw Data Repo updated.

**Comment by cdobs** [comment]
I commented out the lines for PHSEN/PCO2W inductive data because the parsers need to be updated (https://redmine.oceanobservatories.org/issues/15346)

File: GI01SUMO/GI01SUMO_D00009_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/flort1/*flort*.log,GI01SUMO-SBD12-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/hyd2/*hyd*.log,GI01SUMO-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/pco2a/*pco2a*.log,GI01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/wavss/*wavss*.log,GI01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/metbk2/*metbk*.log,GI01SUMO-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00009/cg_data/dcl12/fdchp/*fdchp*.log,GI01SUMO-SBD12-08-FDCHPA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_D00009_ingest.csv:51]: looks good

File: GI05MOAS-GL525/GI05MOAS-GL525_D00003_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-04-CTDGVM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI05MOAS-GL525/GI05MOAS-GL525_D00003_ingest.csv:5]: looks good but just want to verify that the glider # does not need to be inserted at end of filename_mask (merged-from-glider/gi_525*.mrg?  Some older ingest csvs have it, but maybe that was determined to be unnecessary?

File: GI05MOAS-GL538/GI05MOAS-GL538_D00002_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL538/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-GL538-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL538/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-GL538-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL538/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-GL538-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL538/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-GL538-04-CTDGVM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI05MOAS-GL538/GI05MOAS-GL538_D00002_ingest.csv:5]: looks good - no data available on raw data repo yet; same question as above re filename_mask

File: GI05MOAS-PG575/GI05MOAS-PG575_D00003_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG575/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG575-02-DOSTAM000,telemetered,Available,Short deployment
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG575/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG575-03-FLORTM000,telemetered,Available,Short deployment
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG575/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG575-04-FLORTO000,telemetered,Available,Short deployment
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG575/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG575-05-NUTNRM000,telemetered,Available,Short deployment
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG575/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG575-06-PARADM000,telemetered,Available,Short deployment
```
Comment (rtravis-whoi) [inline GI05MOAS-PG575/GI05MOAS-PG575_D00003_ingest.csv:8]: looks good - no data available on raw data repo yet; same comment as above re filename_mask

File: GP05MOAS-GL363/GP05MOAS-GL363_D00006_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,telemetered,Available,A 3-wavelength fluorometer (FLORTM) was deployed in place of a 2-wavelength (FLORDM). This is the parser for 3-wavelength fluorometer
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-04-CTDGVM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GP05MOAS-GL363/GP05MOAS-GL363_D00006_ingest.csv:5]: looks good; confirmed via deploy.csv that for the FLORTM in place of FLORDM the refdes is correct (stays as GP05MOAS-GL363-01-FLORDM000)
Same comment as above re filename_mask.

**Review by rtravis-whoi — APPROVED** [review]
looks good but just want to verify that the glider # does not need to be inserted at end of filename_mask (for example: merged-from-glider/gi_525*.mrg vs merged-from-glider/gi_*.mrg). Some older ingest csvs have it, but maybe that was determined to be unnecessary?

**Pull request merged**